src/csvReaderWrapper.js

```diff
@@ -14,12 +18,37 @@ export const parseCSV = (csvfile, download, setColumnsToOther) => {
   });
 };
 
+const isCellValid = (cell) => {
+  return cell !== undefined && cell !== "";
+}
+
+const isRowValid = (row) => {
+  var cells = Object.values(row);
+  for (var i = 0; i < cells.length; i++) {
+    if (!isCellValid(cells[i])) {
+      return false;
+    }
+  }
```

Comment:
I think we could use `cells.every` here?

Reply:
Well, isn't that tidy? Thanks!